README.rst

```diff
@@ -144,17 +144,17 @@ connect to your chosen backend (XMPP, Hipchat, Slack ...) server.
 The first time you start Err, it is recommended to run it in foreground mode. This can
 be done with::
 
-    <path_to_install_directory>/scripts/err.py
+    <path_to_install_directory>/errbot/err.py
```

Comment:
maybe we can encourage people to do the cleaner `pip install -e .` instead and execute `errbot` ?